NEWS

```diff
@@ -218,14 +396,91 @@ Noteworthy changes in release a.b
   Note that while BCF files affected by this bug are technically incorrect,
   it is still possible to read them.  When converting to VCF format,
   HTSlib (and therefore bcftools) will interpret the values as intended
-  and write out the correct negative numbers. (#766, thanks to John Marshall)
+  and write out the correct negative numbers. (#766, thanks to John Marshall;
+  samtools/bcftools#874)
+
+* Allow repeated invocations of bcf_update_info() and format_*()
```

Comment:
Suggestions almost applied :smile: (Original was 8ab9acf01b5fc95c2a6c18ace1e0a489d0089261 “Allow repeated invocations of bcf_update_info/format_*()” but with `/` replaced by `and` it probably needs the full function name prefix for clarity.

```suggestion
* Allow repeated invocations of bcf_update_info() and bcf_update_format_*()
```

Reply:
Should be right now.

Reply:
:+1: thanks